### PR TITLE
Refactor CLI Install Script

### PR DESCRIPTION
**Pull request by Hazealign**
- Check Node.js Minimum Version (Above v8)
 - Add Error Handler in Download
 - Use `util.promisify` and split promises for refactor.

### CODE REVIEW

**Comment by riemannulus** [comment]
Thanks for your contribution! Can you add [skip changelog] to your commit message to bypass changelog checking?

File: Libplanet.Tools/download.js [hunk 1]
```diff
@@ -18,64 +24,89 @@ const SUFFIXES = {
...
+    };
+
+    if (log != null) log(`
 -------------------------------------------------------------------------------
 Downloaing Libplanet CLI Tools ${ver} from GitHub...
-
 NOTE: You can avoid npm entirely by downloading directly from:
-
   ${url}
-
```
Comment (dahlia) [inline Libplanet.Tools/download.js:73]: These empty lines between statements were for readability; could you maintain them?

**Review by dahlia — COMMENTED** [review]
Thanks for your first contribution!

**Review by riemannulus — COMMENTED** [review]
👍

**Review by riemannulus — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Review by longfin — APPROVED** [review]
👍

**Pull request merged**